README.md

```diff
@@ -6,27 +6,26 @@
 
 This action checks-out your repository under `$GITHUB_WORKSPACE`, so your workflow can access it.
```

Comment:
The existing summary does not have enough details and folks are getting tripped up by the fetch-depth 1 default